CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+## v0.24.0 (2024-01-10)
+
+## What's Changed
+
+Release highlights:
+* IP colocation filter can now be changed via a configuration parameter.
+* New filter admin endpoint can now be used to access subscription data.
+
+### Features
+
+- feat: adding filter data admin endpoint (REST) [#2314](https://github.com/waku-org/nwaku/pull/2314)
+- ip colocation is parameterizable. If set to 0, it is disabled [#2323](https://github.com/waku-org/nwaku/pull/2323)
+
+### Bug Fixes
+
+- fix: Set record to the Waku Node Builder in the examples as it is required [#2328](https://github.com/waku-org/nwaku/pull/2328)
+- fix(discv5): add bootnode filter exception [#2267](https://github.com/waku-org/nwaku/pull/2267)
+
+### Changes
+
+- chore: Update CHANGELOG.md to reflect bug fix for issue [#2317](https://github.com/waku-org/nwaku/issues/2317) [#2340](https://github.com/waku-org/nwaku/pull/2340) in v0.23.1
+- test(peer-connection-managenent): Functional Tests [#2321](https://github.com/waku-org/nwaku/pull/2321) @ e9d083b introduced build error through ambigous function call, testwaku build failed on master [#2337](https://github.com/waku-org/nwaku/pull/2337)
+- docs: update after release steps [#2336](https://github.com/waku-org/nwaku/pull/2336)
+- docs: Fix Typos Across Various Documentation Files [#2310](https://github.com/waku-org/nwaku/pull/2310)
+- test(peer-connection-managenent): Functional Tests [#2321](https://github.com/waku-org/nwaku/pull/2321)
+- bump vendors for 0.24.0 [#2333](https://github.com/waku-org/nwaku/pull/2333)
+- test(autosharding): Functional Tests [#2318](https://github.com/waku-org/nwaku/pull/2318)
+- docs: add benchmar around postgres adoption [#2316](https://github.com/waku-org/nwaku/pull/2316)
+- chore: message.nim - set max message size to 150KiB according to spec [#2298](https://github.com/waku-org/nwaku/pull/2298)
```

Comment:
Should this be an upgrade note? In other words, those running nwaku and want to continue supporting the "old" message size of 1MB now needs to explicitly configure this?

Reply:
Yes true!